src/main/kotlin/uk/gov/justice/probation/courtcaseservice/controller/CaseWorkflowController.kt

```diff
@@ -15,16 +15,21 @@ import javax.validation.Valid
 @RestController
 class CaseWorkflowController(val caseWorkflowService: CaseWorkflowService, val authenticationHelper: AuthenticationHelper) {
 
-    @Operation(description = "Adds hearing outcome for a hearing.")
+    @Operation(description = "Adds or updates hearing outcome for a hearing.")
     @PutMapping(value = ["/hearing/{hearingId}/outcome"], produces = [APPLICATION_JSON_VALUE], consumes = [APPLICATION_JSON_VALUE])
     fun addHearingOutcome(@PathVariable("hearingId") hearingId: String, @RequestBody hearingOutcome: HearingOutcome) =
```

Comment:
Please change the name to addOrUpdateHearingOutcome or upsertHearingOutCome.

Reply:
Will update, thanks

Reply:
Updated @vb-moj